platform/security/common/src/main/java/org/codice/ddf/security/common/jaxrs/RestSecurity.java

```diff
@@ -50,6 +50,8 @@
 
   public static final boolean GZIP_COMPATIBLE = true;
 
+  private RestSecurity() {}
```

Comment:
:question: Why are we making a class instantiation here?

Reply:
It's a class that only contains static methods so by adding that private constructor you "hide" the default public one.